pkgs/node/src/clock.zig

```diff
@@ -64,15 +95,9 @@ pub const Clock = struct {
         }
     }
 
-    pub fn onSlot() void {}
+    pub fn subscribeOnSlot(self: *Self, cb: *OnSlotCbWrapper) !void {
+        try self.on_slot_cbs.append(cb);
+        var c: xev.Completion = undefined;
+        try self.on_slot_cs.append(&c);
```

Comment:
I don't know `xev.Completion` but I don't think that will work: you are passing a pointer to a stack-allocated object,